### PR TITLE
Appaloosa store #2

### PR DESCRIPTION
**Pull request by yyaegash**
# Introduction

Appaloosa is a private mobile application store. This action offers a quick deployment on the platform.
You can create an account, push to your existing account, or manage your user groups.
We accept iOS and Android applications.
## Variables used

We use xcodebuild to build your .ipa, and Snapshot to load your screenshots. All their variables are reused by default. If you use other actions, we will ask you for missing information.
# Usage

``` ruby
default_platform :ios

platform :ios do

    lane :scenario_0 do # context: signup and publish. A screenshot folder path will be asked
        gym(use_legacy_build_api: true)
        appaloosa
    end

    lane :scenario_1 do # context: user used snapshot with specific values.
        snapshot 
        gym(use_legacy_build_api: true)
        appaloosa(
            store_id: 'your_store_id',
            api_token: 'your_api_key',
            locale: 'de-DE', # When multiple values are specified in the Snapfile, we default to 'en-US'.
            device: 'iPhone6' # By default, the screenshots from the last device will be used.
        }
    end

    lane :scenario_2 do # context: user specifies some groups.
        appaloosa(
            binary: 'your_binary', # your ipa or apk
            store_id: 'your_store_id',
            api_token: 'your_store_id',
            group_ids: '1, 2, 3,..'
        )
    end
end
```

### CODE REVIEW

File: docs/Guide.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Requirements:
 Additionally, to an Xcode installation, you also need the Xcode command line tools set up
 
     xcode-select --install
+    
+If you have not used the command line tools before (which is likely if you just installed it), you'll need to accept the terms of service.
```
Comment (KrauseFx) [inline docs/Guide.md]: That shouldn't be added with this pull request, please remove this change.

File: lib/fastlane/actions/appaloosa.rb [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  module Actions
+    class AppaloosaAction < Action
+      APPALOOSA_SERVER = 'http://appaloosa-int.herokuapp.com/api/v1'
+      def self.run(params)
+        require 'http'
+
+        api_key = params[:api_token]
+        store_id = params[:store_id]
+
+        if request_email? api_key, store_id
```
Comment (KrauseFx) [inline lib/fastlane/actions/appaloosa.rb]: Please use `(` around the parameters

File: lib/fastlane/actions/appaloosa.rb [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+          auth = create_an_account params[:email]
```
Comment (KrauseFx) [inline lib/fastlane/actions/appaloosa.rb]: Same here and on below code.

**Comment by KrauseFx** [comment]
Thanks for the pull request. I added some comments. As I didn't really test the integration, can you provide ongoing support if a user reports an issue? If so, how can a user contact you in case of problems? If you prefer email, feel free to add your email to the `self.details` output.

File: lib/fastlane/actions/appaloosa.rb [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Upload your app to Appaloosa Store'
+      end
+
+      def self.details
+        'You can use this action to do cool things...'
```
Comment (KrauseFx) [inline lib/fastlane/actions/appaloosa.rb]: Please change this to an actual description on what this action does.

**Comment by jeremybdk** [comment]
All done, let me know if you have more questions

**Comment by KrauseFx** [comment]
Hey, really sorry for the delay from my side. This is a great addition. Could you add the most minimalistic example to the `Actions.md` file?

**Pull request merged**